website/content/60-advanced/601-emr-integration/6014-verify-access/60141-zeppelin.md

```diff
@@ -19,13 +19,19 @@ autoNav = true
       <li>Clear the browser cache/cookies as your previous login into IdP account is still in the session <b>OR</b> open in Incognito mode if you are using Google Chrome.</li>
       <li>To access Apache Zeppelin, copy the <b>EMRMasterNodeDNS</b> value from CloudFormation stack output. Using your browser, navigate to the following URL. Ensure the URL includes the trailing slash at the end and updated EMRMasterNodeDNS address.<pre>https://<b>EMRMasterNodeDNS</b>:8442/gateway/default/zeppelin/</pre></li>
       <li>Once the Proxy Agent’s certificate is accepted, your browser redirects you to your Identity Provider (IdP) login page to authenticate.
+         <ul>
+            <li>For Auth0/Okta use <b>emr-developer@somecompany.com</b> and password you provided to authenticate</li>
+            <li>For AD FS use <b>emr-developer@hadoop.com</b> and password (Password1!) to authenticate</li>
+         </ul>
+
          <div class="row">
             <div class="column"><img src="/images/auth0-authenticate.png" style="margin:15px 0px; border:1px solid black"/></div>
             <div class="column"><img src="/images/okta-login.png" style="margin:15px 0px; border:1px solid black"/></div>
+            <div class="column"><img src="/images/adfs-login.png" style="margin:15px 0px; border:1px solid black"/></div>
          </div>
```

Comment:
Reformat this section, it shows two boxes in one row and one large box in 2nd row

Reply:
changed the  css style .column to 33.33%